lib/lhm/migrator.rb

```diff
@@ -210,6 +210,7 @@ def destination_create
       original    = %{CREATE TABLE `#{ @origin.name }`}
       replacement = %{CREATE TABLE `#{ @origin.destination_name }`}
       stmt = @origin.ddl.gsub(original, replacement)
+        .gsub(/DEFAULT CHARSET=\w*/, "DEFAULT CHARSET=utf8mb4 ROW_FORMAT=DYNAMIC")
```

Comment:
Is `DEFAULT CHARSET=` _always_ part of the DDL? If so, this will certainly work. If not, then you probably need some more code here that adds it to the DDL if it's not present.